third_party/move/move-binary-format/src/file_format.rs

```diff
@@ -886,6 +886,15 @@ pub struct AccessSpecifier {
     pub address: AddressSpecifier,
 }
 
+impl AccessSpecifier {
+    pub fn is_traditional_acquires(&self) -> bool {
```

Comment:
Check whether an access specifier is the traditional one. This check is used when serializing the function handle.